tools/hermes-parser/js/flow-api-translator/src/flowToFlowDef.js

```diff
@@ -977,6 +978,35 @@ function convertClassDeclaration(
   ];
 }
 
+function convertExpressionToIdOrQualifiedTypeId(
+  node: Expression,
+  context: TranslationContext,
+): DetachedNode<Identifier> | DetachedNode<QualifiedTypeIdentifier> {
+  if (node.type === 'Identifier') {
+    return t.Identifier({name: node.name});
+  } else if (
+    node.type === 'MemberExpression' &&
+    node.property.type === 'Identifier' &&
+    node.object.type !== 'Super'
+  ) {
+    const id = t.Identifier({name: node.property.name});
+    return t.QualifiedTypeIdentifier({
+      qualification: convertExpressionToIdOrQualifiedTypeId(
+        // $FlowFixMe[incompatible-call] conditional already checks for non-Super type
```

Comment:
not sure if I addressed this the right way, tried to convert the MemberExpression but hit this snag with Super type.

Reply:
This seems practical to me, but prefer `// $FlowIssue` (indicating the typechecker should have been able to handle this, and it's not a fixme on this code).

```suggestion
        // $FlowIssue[incompatible-call] conditional already checks for non-Super type
```